src/Uno.UI/UI/Xaml/WindowManagerInterop.Wasm.cs

```diff
@@ -614,6 +694,81 @@ private struct WindowManagerSetContentHtmlParams
 			public string Html;
 		}
 
+		#endregion
+
+		#region ArrangeElement
+
+		internal static void ArrangeElement(IntPtr htmlId, Rect rect, Rect? clipRect)
+		{
+			if (UseJavascriptEval)
+			{
+				var clipRect2 = clipRect != null
+					? string.Format(
+						CultureInfo.InvariantCulture,
+						"rect({0}px,{1}px,{2}px,{3}px",
+						clipRect.Value.Top,
+						clipRect.Value.Right,
+						clipRect.Value.Bottom,
+						clipRect.Value.Left
+					)
+					: "";
+
+				SetStyles(
```

Comment:
Using this won't remove the class `uno-unarranged` anymore... leading to showing nothing.

Reply:
That's what the `true` below is for.